### PR TITLE
Add manual pages for current commands

### LINKED ISSUE #3: man pages for dotnet
dotnet executable should support printing its own usage instructions. The same should be printed when calling 'man dotnet'

### PR DESCRIPTION
**Pull request by blackdwarf**
Convert the existing README.md from command directories into man pages. This
was done using pandoc. Some commands still don't have READMEs so they are not
present here. Also change the format of the README.md files to be more amicable to 
manpages transformation.

Fix #3

/cc @piotrpMSFT @brthor @anurse

### CODE REVIEW

**Comment by brthor** [comment]
@blackdwarf Would it be preferable to add this as an automated step to the build, rather than checking these in and needing to maintain the synchronization between the .md files?

**Comment by blackdwarf** [comment]
@brthor they would, this is an interim step to have them right now for testing out the actual formatting and stuff. I will take a look at the build def next.

**Comment by brthor** [comment]
Makes sense. 

A step needs to be added to the build to include these manpages in the debian package. I will make a quick PR to your branch.

**Comment by borgdylan** [comment]
With regards to Linux packaging, make sure to have these installed to ```/usr/share/man/man1```

**Comment by brthor** [comment]
@borgdylan Using the package tool (https://github.com/dotnet/cli/tree/master/packaging/debian/package_tool) these manpages will be automatically installed using `dh_installman`

http://man.he.net/man1/dh_installman

File: Documentation/manpages/dotnet-cli.1 [hunk 1]
```diff
@@ -0,0 +1,34 @@
+.\" Automatically generated by Pandoc 1.15.1
```
Comment (brthor) [inline Documentation/manpages/dotnet-cli.1]: This file should be named "dotnet.1" to enable `man dotnet`

File: Documentation/manpages/dotnet-native.1 [hunk 2]
```diff
@@ -0,0 +1,57 @@
+.\" Automatically generated by Pandoc 1.15.1
```
Comment (brthor) [inline Documentation/manpages/dotnet-native.1]: Likewise here, or do we expect users to do `man dotnet-native` ?

**Comment by brthor** [comment]
@blackdwarf Check the PR just made to this branch to include these in the deb package.

File: Documentation/manpages/dotnet-native.1 (hunk 2 above)
Comment (blackdwarf) [inline Documentation/manpages/dotnet-native.1]: This one is tricky, I will leave as-is, but it should become a part of the dotnet-compile man page.

**Pull request merged**